doc/ExpertOptions.md

```diff
@@ -823,11 +876,11 @@ it may misplace or outright delete files. The applet:
 
 ## Adding config-file based reorg applet at compilation time
 
-In addition to using `--reorg` flag to add the reorg stage, you may also add a custom reorganization applet that takes an optional input by declaring a "custom-reorg" object in the JSON file used as parameter with `-extras`
+In addition to using `--reorg` flag to add the reorg stage, you may also add a custom reorganization applet that takes an optional input by declaring a "customReorgAttributes" object in the JSON file used as parameter with `-extras`
```

Comment:
Why introduce a change in syntax?  Will the old syntax still work?  Here and for other customer facing fields below.

Reply:
The problem is inconsistency. Some keys have dashes, some use underscores, and some are camel case. The code still handles all cases so it will not break existing extras.json files.